client/my-sites/email/email-providers-stacked-comparison/provider-cards/provider-card-props.ts

```diff
@@ -0,0 +1,63 @@
+import { Domain } from '@automattic/data-stores/dist/types/site';
+import { TranslateResult } from 'i18n-calypso';
+import { ReactElement } from 'react';
+
+export type Site = {
+	ID: number;
+	slug: string;
+};
```

Comment:
Can we import this from the code that actually defines the interface? It feels like this is not the right place to define this type.

Reply:
I can't understand what you mean. I can't find any other type for Site anywhere... The only place I can find something similar is `import type { Site } from 'calypso/my-sites/scan/types'; `and it does not provide the slug